dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/MultipleApproverActionlet.java

```diff
@@ -64,95 +54,93 @@ public void executeAction(WorkflowProcessor processor, Map<String, WorkflowActio
 			}
 		}
 
-		Set<User> requiredApprovers = new HashSet<User>();
-		Set<User> hasApproved = new HashSet<User>();
-		StringTokenizer st = new StringTokenizer(userIds, ", ");
-		while (st.hasMoreTokens()) {
-			String x = st.nextToken();
+		final Set<User> requiredApprovers = new HashSet<>();
+		final Set<User> hasApproved       = new HashSet<>();
+		final StringTokenizer userIdTokenizer = new StringTokenizer(userIds, StringPool.COMMA);
+		while (userIdTokenizer.hasMoreTokens()) {
 
-			if (Validator.isEmailAddress(x)) {
+			final String userIdToken = userIdTokenizer.nextToken();
+
+			if (Validator.isEmailAddress(userIdToken)) {
 				try {
-					User u = APILocator.getUserAPI().loadByUserByEmail(x, APILocator.getUserAPI().getSystemUser(), false);
 
-					requiredApprovers.add(u);
+					final User user = APILocator.getUserAPI().loadByUserByEmail(userIdToken,
+							APILocator.getUserAPI().getSystemUser(), false);
+
+					requiredApprovers.add(user);
 				} catch (Exception e) {
-					Logger.error(this.getClass(), "Unable to find user with email:" + x);
+
+					Logger.error(this.getClass(), "Unable to find user with email:" + userIdToken);
 				}
 			} else {
 				try {
 
-					User u = APILocator.getUserAPI().loadUserById(x, APILocator.getUserAPI().getSystemUser(), false);
-					requiredApprovers.add(u);
+					final User user = APILocator.getUserAPI().loadUserById(userIdToken,
+							APILocator.getUserAPI().getSystemUser(), false);
+					requiredApprovers.add(user);
 				} catch (Exception e) {
-					Logger.error(this.getClass(), "Unable to find user with userID:" + x);
+					Logger.error(this.getClass(), "Unable to find user with userID:" + userIdToken);
```

Comment:
1. These should be a `Logger.warn`
2. If we are not going to log the stack, which is ok, we should at least include the `e.getMessage()` in our message.  Otherwise the root cause is totally obfuscated.  `Logger.warnAndDebug()` works as well.